src/mbgl/renderer/layers/render_custom_layer.cpp

```diff
@@ -46,11 +47,11 @@ void RenderCustomLayer::render(PaintParameters& paintParameters, RenderSource*)
     if (context != impl().context || !initialized) {
         //If the context changed, deinitialize the previous one before initializing the new one.
         if (context && !contextDestroyed && impl().deinitializeFn) {
-            impl().deinitializeFn(context);
+            MBGL_CHECK_ERROR(impl().deinitializeFn(context));
```

Comment:
Is this going to break on Qt due to the special use of `MBGL_CHECK_ERROR` there? (#11106)

cc @kkaefer @tmpsantos

Reply:
Not really, Qt is not using `MBGL_CHECK_ERROR` as described by https://github.com/mapbox/mapbox-gl-native/pull/11106